pipeline.go

```diff
@@ -104,8 +104,28 @@ func generatePipeline(steps []interface{}, pipelineEnv map[string]string, projec
 
 		env, foundEnv := stepMap["env"].(map[interface{}]interface{})
 		_, foundBlockStep := stepMap["block"].(string)
+		_, foundTriggerStep := stepMap["trigger"].(string)
 
-		if !foundBlockStep {
+		if foundTriggerStep {
```

Comment:
It would be helpful to add a short comment describing the transformation that is being done in this block.